### PR TITLE
Explore more

### PR DESCRIPTION
**Pull request by KonstantinosDimakis**
Please review this, @casey-collab

### CODE REVIEW

File: english/33hours/explore-outside/outside.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+[Start walking east, on the highway](./explore-east.md)
```
Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by KonstantinosDimakis** [comment]
@casey-collab Please review this!

File: english/33hours/explore-outside/outside.md [hunk 2]
```diff
@@ -5,4 +5,6 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+
+[Start walking east, on the highway](/explore-east.md)
```
Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by KonstantinosDimakis** [comment]
@casey-collab Please review this!

File: english/33hours/explore-outside/outside.md [hunk 3]
```diff
@@ -5,4 +5,6 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+
+[Start walking east, on the highway](../explore-room/explore-east.md)
```
Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/33hours/explore-outside/outside.md [hunk 4]
```diff
@@ -5,4 +5,6 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+
+[Start walking east, on the highway](create-your-own-adventure/english/33hours/explore-outside/explore-east.md)
```
Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by KonstantinosDimakis** [comment]
@casey-collab Please review this!

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/33hours/explore-outside/outside.md (hunk 4 above)
Comment (casey-collab) [inline english/33hours/explore-outside/outside.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by KonstantinosDimakis** [comment]
@casey-collab please review this

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by KonstantinosDimakis** [comment]
@casey-collab review this please!

**Comment by casey-collab** [comment]
It looks like the file `english/33hours/explore-outside/explore-outside-east/explore-east.md` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by KonstantinosDimakis** [comment]
@casey-collab Please review this

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**